agent/battle/perception/config.py

```
@@ -0,0 +1,54 @@
+
+识别节点定义在 assets/resource/base/pipeline/自动战斗_感知.json：
+  - 战斗_选卡场景        TemplateMatch（选卡界面特征）
+  - 战斗_主界面          TemplateMatch（主界面攻击钮）
+  - 战斗_胜利 / 战斗_失败   OCR / TemplateMatch
+  - 战斗_卡{1..5}        OCR（识别「力击/迅击/技击」）
+  - 战斗_NP卡{1..3}      OCR（识别 NP 数值，≥100% 可用）
+  - 战斗_敌人{1..3} / 战斗_敌人{1..3}_选中
+"""
+from __future__ import annotations
+
+# 场景检测节点（按序尝试，先命中者定场景）
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)